src/components/repository-list-item.component.js

```diff
@@ -63,7 +63,7 @@ const renderTitle = repository =>
     <View style={styles.repositoryContainer}>
       <View style={styles.titleWrapper}>
         <Text style={styles.title}>
-          {repository.full_name}
+          {repository.name}
```

Comment:
As correctly noted, displaying only the name of the repository will not be convenient, so we need to add a check: if this is an auth screen, then show only the name of the repository, and if the user profile, then display the full name.

Reply:
Although, maybe it can just add the `lock` icon to the end of the title? 
@chinesedfan so in fact will work?

Reply:
Yes, it doesn't matter the pending element is the private box or the lock icon, as long as the parent element is Text and the child view has width&height specified.

Reply:
Done